hr_expense_tag/models/hr_expense_sheet_tag.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 ForgeFlow
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from random import randint
+from odoo import fields, models
+
+class Tag(models.Model):
+    _name = "hr.expense.sheet.tag"
+
+    _description = "Expense sheet Tag"
+
+    def _get_default_color(self):
+        return randint(1, 11)
+
+    name = fields.Char('Tag Name', required=True, translate=True)
+    color = fields.Integer('Color', default=_get_default_color)
+    _sql_constraints = [
+        ('name_uniq', 'unique (name)', "Tag name already exists !"),
+    ]
```

Comment:
```suggestion
```
Maybe we could only add the `hr.expense.tag` model, instead of having two very similar models and then use the model in both the expense and the expense sheet. WDYT?